Mathlib/Topology/Algebra/Module/ModuleTopology.lean

```diff
@@ -492,4 +494,109 @@ instance instPi : IsModuleTopology R (∀ i, A i) := by
 
 end Pi
 
+section bilinear
+
+section semiring
+
+variable {R : Type*} [TopologicalSpace R] [CommSemiring R]
+variable {B : Type*} [AddCommMonoid B] [Module R B] [TopologicalSpace B] [IsModuleTopology R B]
+variable {C : Type*} [AddCommMonoid C] [Module R C] [TopologicalSpace C] [IsModuleTopology R C]
+
+/--
+If `ι` is finite and `B`,`C` are `R`-modules with the module topology,
+then any bilinear map `Rᶥ × B → C` is automatically continuous.
```

Comment:
```suggestion
then any bilinear map `Rᶥ × B → C` is automatically continuous.
```
Can you replace `ᶥ` by `I` or something? The doc is quite hard to read, it seems `R'` (`R` prime).

Reply:
If I go for "If `n` is finite" then we can use `Rⁿ` which is much easier to recognise.